aequilibrae/paths/linear_approximation.py

```diff
@@ -325,6 +329,15 @@ def execute(self):
                 # cost = c.fixed_cost / c.vot + self.congested_time #  now only once
                 cost = c.fixed_cost + self.congested_time
                 aggregate_link_costs(cost, c.graph.compact_cost, c.results.crosswalk)
+
+                if c._aon_results.save_path_file:
+                    # TODO Jan: make base dir user configurable
+                    pth = os.environ.get(ENVIRON_VAR)
+                    path_base_dir = os.path.join(pth, "path_files", self.procedure_id, f"iter{self.iter}")
+                    # TODO Jan: what identifier do we use for the class?
+                    c._aon_results.path_file_dir = os.path.join(path_base_dir, f"path_c{c.mode}")
```

Comment:
```suggestion
                    c._aon_results.path_file_dir = os.path.join(path_base_dir, f"path_c{c.mode}_{c.__id__}")
```

Using mode only leads us to that issue we had before of multiple classes using the same road mode. We could think about another way of differentiating the classes (tying to the matrices, maybe), but for now, we should have the ID there

Reply:
true, forgot about that